Tests/Scenarios/Tentacle_Scenario_03_Reinstall.ps1

```diff
@@ -31,8 +31,8 @@ Configuration Tentacle_Scenario_03_Reinstall
             # Optional settings
             ListenPort = 10933;
             DefaultApplicationDirectory = "C:\Applications"
-            TentacleDownloadUrl = "https://download.octopusdeploy.com/octopus/Octopus.Tentacle.3.3.24.msi"
-            TentacleDownloadUrl64 = "https://download.octopusdeploy.com/octopus/Octopus.Tentacle.3.3.24-x64.msi"
+            TentacleDownloadUrl = "https://download.octopusdeploy.com/octopus/Octopus.Tentacle.3.20.0.msi"
```

Comment:
Let's add an issue to this repo to revert this

Reply:
done